### ISSUE TITLE
Add the Ability to DM with Users

### ISSUE THREAD

**Issue by jellyjam14blas** [nedius/discord4bot#13]
Hello

This client has the ability to Message in Server channels, but can't read/send user's Direct Messages, which is a shame. 
I think it would help round out the client.

Have a wonderful day! <3
- Jelly

**Comment by nedius** [comment]
Yes I know about that. It's planned feature. But there is some interesting behaviour. You can send dm and receive, but you can't fetch previous dms.
For now I'm rewriting discord4bot to Vue.
